sync2/atxs.go

```diff
@@ -197,14 +197,26 @@ func NewMultiEpochATXSyncer(
 	hss HashSyncSource,
 	oldCfg, newCfg Config,
 	parallelLoadLimit int,
-) *MultiEpochATXSyncer {
+) (*MultiEpochATXSyncer, error) {
+	var err error
+	if vErr := oldCfg.Validate(); vErr != nil {
+		// extra newline added to the error message to make it more
+		// readable, as it may contain multiple config errors
+		err = errors.Join(err, fmt.Errorf("old epoch config:\n%w", vErr))
+	}
+	if vErr := newCfg.Validate(); vErr != nil {
+		err = errors.Join(err, fmt.Errorf("new epoch config:\n%w", vErr))
+	}
+	if err != nil {
+		return nil, err
+	}
```

Comment:
Based on my previous comment this would just become:

```suggestion
	if !oldCfg.Validate(logger) || !newCfg.Validate(logger) {
		return nil, errors.New("invalid config")
	}
```

Reply:
Done